src/stats.js

```diff
@@ -55,84 +51,89 @@ export default async function stats(options = DEFAULT_OPTIONS, context, next) {
      * From this point onward, a valid uncached response has been received.
      */
 
-    context.res.stats.requestCount += 1;
+    context.res.stats.metrics.requestCount += 1;
 
     // invalid HTTP response
     if (context.res.statusCode < 100) {
-      context.res.stats.responseInvalidCount += 1;
+      context.res.stats.metrics.responseInvalidCount += 1;
```

Comment:
I'm totally nitpicking here but I wonder if this could / should be less verbose. (My go-to would be `fp.cond` but I see we don't have lodash here.) Perhaps a little function that maps http codes to metric labels, this code then just says 'find the relevant metric and increment by 1'? Like I said, I'm nitpicking. =)

Reply:
Absolutely, it can be refactored in a helper function 👍